codebase2/util-term/U/Util/Type.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS_GHC -Wno-forall-identifier #-}
```

Comment:
Maybe just `s/forall/forAll` as recommended by the manual instead?

https://downloads.haskell.org/ghc/latest/docs/users_guide/using-warnings.html?highlight=forall%20identifier#ghc-flag--Wforall-identifier

Reply:
Yeah I should've called it out better, this is just a temporary measure since it will become mandatory soon. `forAll` sounds like a good fix.